proposals/simd/webassembly-opcodes.md

```diff
@@ -0,0 +1,275 @@
+| `v32x4.store1(addr, offset, data: v128)` | [v32x4.store1](portable-simd.md#partial-store) |
+| `v32x4.store2(addr, offset, data: v128)` | [v32x4.store2](portable-simd.md#partial-store) |
+| `v32x4.store3(addr, offset, data: v128)` | [v32x4.store3](portable-simd.md#partial-store) |
+
+## `i32x4` operations
+| WebAssembly | Portable SIMD |
+|:------------|:--------------|
+| `i32x4.build(x: i32[4]) -> v128` | [i32x4.build](portable-simd.md#build-vector-from-individual-lanes) |
+| `i32x4.splat(x: i32) -> v128` | [i32x4.splat](portable-simd.md#create-vector-with-identical-lanes) |
+| `i32x4.extractLane(a: v128, i: LaneIdx4) -> i32` | [i32x4.extractLane](portable-simd.md#extract-lane-as-a-scalar) |
+| `i32x4.replaceLane(a: v128, i: LaneIdx4, x: i32) -> v128` | [i32x4.replaceLane](portable-simd.md#replace-lane-value) |
+| `i32x4.add(a: v128, b: v128) -> v128` | [i32x4.add](portable-simd.md#integer-addition) |
+| `i32x4.sub(a: v128, b: v128) -> v128` | [i32x4.sub](portable-simd.md#integer-subtraction) |
+| `i32x4.mul(a: v128, b: v128) -> v128` | [i32x4.mul](portable-simd.md#integer-multiplication) |
+| `i32x4.neg(a: v128) -> v128` | [i32x4.neg](portable-simd.md#integer-negation) |
+| `i32x4.shl(a: v128, y: i32) -> v128` | [i32x4.shiftLeftByScalar](portable-simd.md#left-shift-by-scalar) |
+| `i32x4.shr_s(a: v128, y: i32) -> v128` | [s32x4.shiftRightByScalar](portable-simd.md#right-shift-by-scalar) |
+| `i32x4.shr_u(a: v128, y: i32) -> v128` | [u32x4.shiftRightByScalar](portable-simd.md#right-shift-by-scalar) |
+| `i32x4.eq(a: v128, b: v128) -> b32x4` | [i32x4.equal](portable-simd.md#equality) |
+| `i32x4.ne(a: v128, b: v128) -> b32x4` | [i32x4.notEqual](portable-simd.md#non-equality) |
+| `i32x4.lt_s(a: v128, b: v128) -> b32x4` | [s32x4.lessThan](portable-simd.md#less-than) |
+| `i32x4.lt_u(a: v128, b: v128) -> b32x4` | [u32x4.lessThan](portable-simd.md#less-than) |
+| `i32x4.le_s(a: v128, b: v128) -> b32x4` | [s32x4.lessThanOrEqual](portable-simd.md#less-than-or-equal) |
+| `i32x4.le_u(a: v128, b: v128) -> b32x4` | [u32x4.lessThanOrEqual](portable-simd.md#less-than-or-equal) |
+| `i32x4.gt_s(a: v128, b: v128) -> b32x4` | [s32x4.greaterThan](portable-simd.md#greater-than) |
+| `i32x4.gt_u(a: v128, b: v128) -> b32x4` | [u32x4.greaterThan](portable-simd.md#greater-than) |
+| `i32x4.ge_s(a: v128, b: v128) -> b32x4` | [s32x4.greaterThanOrEqual](portable-simd.md#greater-than-or-equal) |
+| `i32x4.ge_u(a: v128, b: v128) -> b32x4` | [u32x4.greaterThanOrEqual](portable-simd.md#greater-than-or-equal) |
+| `i32x4.trunc_s/f32x4(a: v128) -> v128` | [s32x4.fromFloat](portable-simd.md#floating-point-to-integer) |
+| `i32x4.trunc_u/f32x4(a: v128) -> v128` | [u32x4.fromFloat](portable-simd.md#floating-point-to-integer) |
+
+## `f32x4` operations
+| WebAssembly | Portable SIMD |
+|:------------|:--------------|
+| `f32x4.build(x: f32[4]) -> v128` | [f32x4.build](portable-simd.md#build-vector-from-individual-lanes) |
```

Comment:
We talked about this for asm.js too and ended up keeping them. I'll file an issue.